sipXyealink/etc/yealink/types.xml

```diff
@@ -200,7 +201,7 @@
 </type>
 <type id="InfoType_type">
     <enum>
-	<option><label>Disabled</label><value>0</value></option>
+	<!--<option><label>Disabled</label><value>0</value></option>-->
```

Comment:
Ok, on V73_40, values are:
Configures the DTMF type for account X.
0-INBAND
1-RFC 2833
2-SIP INFO
3-RFC2833 + SIP INFO,
So we should use a different type for V8X? If that's ok for you I will do that.